pkg/sbom/generator/spdx/spdx_test.go

```diff
@@ -79,8 +78,7 @@ func TestReproducible(t *testing.T) {
 		require.NoError(t, err)
 		d = append(d, data)
 	}
-	diff := cmp.Diff(d[0], d[1])
-	require.Empty(t, diff)
+	require.Equal(t, d[0], d[1])
```

Comment:
```suggestion
	diff := cmp.Diff(d[0], d[1])
	require.Empty(t, diff, fmt.Sprintf("difference in expected output %s", diff))
```

Reply:
I'm probably missing something here, testify should already print the diff between "expected" and "got". I tried failing the test on purpose and the only difference I could see was `go-cmp` omitting the identical lines. Did I miss something ?